pkgs/top-level/lua-packages.nix

```diff
@@ -132,6 +132,39 @@ with self; {
     };
   };
 
+  pulseaudio = buildLuaPackage rec {
+    pname = "pulseaudio";
+    version = "0.1";
+    name = "pulseaudio-${version}";
```

Comment:
this line is not needed, since it gets created automatically by `pname` and `version`

Reply:
Yea I thought so too as well but I got an error about this attribute missing:

```
error: attribute 'name' missing, at /var/src/nixpkgs/pkgs/development/lua-modules/generic/default.nix:20:40
```

Well I guess it's an issue with `buildLuaPackage` but I'm not sure exactly where to fix it.

Reply:
hmm, you are indeed right. So I guess we can leave it at that and fix `buildLuaPackage` at a later point.